src/main.py

```diff
@@ -68,13 +70,21 @@ def main(geo_ls=["zip", "county"]):
     and save jsons to file
     '''
     # from src import dict_merge
+
+    print("Reading WIC Data")
+    wic_start_time = time.time()
+    src.wic.read_wic_data()
+    print("Reading WIC Data took: {0:.2f} seconds".format(time.time() - wic_start_time))
```

Comment:
Do we want this sort of logging to be called from main(), or should it be done within the specific script?
Do we want to add options to turn verbose logging on or off?

Reply:
Yeah, I don't think it should be on by default